app/serializers/brack_serializer.rb

```diff
@@ -0,0 +1,10 @@
+require 'brackr/distance_calculator'
+
+class BrackSerializer < ActiveModel::Serializer
+  include Brackr::Distance
```

Comment:
Ooo! So I tried this out and am having some trouble... I'm going to leave it for now, but will keep it in mind as a possible refactor.